restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -151,23 +152,23 @@
 	"externalServiceStepIgnoredParamsWarning": "Parámetros de servicio no se utilizan en las solicitudes de POST/PUT por defecto. Verifique 'Rellenar automáticamente de los parámetros del servicio' para alterar este comportamiento",
 	"externalServiceStepUsernameTooltip": "Nombre de usuario para la autenticación HTTP BASIC. Déjelo en blanco si no se utiliza la autenticación.",
 	"externalServiceStepUsernameLabel": "Usuario",
-	"externalServiceStepPasswordTooltip": "Contraseña para la autenticación HTTP BASIC. Déjelo en blanco si no se utiliza la autenticación.",
+	"externalServiceStepPasswordTooltip": "Contraseña para la autenticación HTTP BASIC. Dejar en blanco si no se utiliza la autenticación.",
 	"externalServiceStepPasswordLabel": "Contraseña",
 	"externalServiceStepFormErrorLabel": "Especifique una dirección URL válida que devuelve JSON",
 	"externalServiceStepUrlParamNameLabel": "Nombre",
 	"externalServiceStepUrlParamValueLabel": "Valor",
 	"externalServiceStepAddParamButton": "Agregar parámetro de servicio.",
-	"externalServiceStepAddParamButtonPopover": "Añadir un parámetro de servicio nombre-valor. Dependiendo de los parámetros del método y de tipo de contenido se colocan en la consulta o el cuerpo de la solicitud",
+	"externalServiceStepAddParamButtonPopover": "Añadir un parámetro de servicio nombre-valor. Dependiendo de los parámetros de método y de tipo de contenido se colocan en la consulta o el cuerpo de la solicitud",
 	"externalServiceStepAssignResponseLabel": "Asigne respuesta a variables",
-	"externalServiceStepAssignResponseText1": "Asigne campos de respuesta JSON a las variables RVD para accesar a los datos devueltos a partir de componentes RVD. Crear una asignación para cada parte separada de información que necesite.",
+	"externalServiceStepAssignResponseText1": "Asigne campos de respuesta JSON a las variables RVD para accesar a los datos devueltos a partir de componentes RVD. Crear una asignación para cada parte separada de información que se necesite.",
 	"externalServiceStepAssignResponseText2": "Para cada asignación:",
-	"externalServiceStepAssignResponseText3": "Llene el campo 'Asignar a' con el nombre de la variable RVD que desea crear.",
+	"externalServiceStepAssignResponseText3": "Llene el campo \"Asignar a' con el nombre de la variable RVD que desea crear.",
 	"externalServiceStepAssignResponseText4": "Haga clic en 'Agregar operación' para añadir una o más operaciones de acceso hasta que llegue a la variable miembro JSON que necesite. Para los objetos utilice la acción 'propertyNamed' para accesar a una propiedad específica. Para las matrices utilice la acción 'itemAtPosition' para acceder al elemento N de la matriz.",

```

Comment:
Grammar error found. Change for:
"externalServiceStepAssignResponseText4": "Haga clic en 'Agregar operación' para añadir una o más operaciones de acceso hasta que llegue a la variable miembro JSON que necesite. Para los objetos utilice la acción 'propertyNamed' para acceder a una propiedad específica. Para las matrices utilice la acción 'itemAtPosition' para acceder al elemento N de la matriz.",